pytorch_lightning/utilities/device_parser.py

```diff
@@ -231,8 +231,8 @@ def _check_unique(device_ids: List[int]) -> None:
 
 
 def _check_data_type(device_ids: Any) -> None:
-    """Checks that the device_ids argument is one of: None, Int, String or List. Raises a MisconfigurationException
-    otherwise.
+    """Checks that the device_ids argument is one of: None, int, string, or sequence of integers. Raises a
+    MisconfigurationException otherwise.
```

Comment:
```suggestion
    """Checks that the device_ids argument is one of None, int, string, or sequence of integers.
```
Misconfiguration details are mentioned below